android/res/color/preference_summary_text.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
don't forget to add the license headers, and also include yourself as one of the authors.